common/djangoapps/student/admin.py

```diff
@@ -147,21 +151,16 @@ class Meta(object):
         model = CourseEnrollment
 
 
-class UserProfileAdmin(admin.ModelAdmin):
-    """ Admin interface for UserProfile model. """
-    list_display = ('user', 'name',)
-    raw_id_fields = ('user',)
```

Comment:
Is getting rid of this raw ID field going to force the inline admin to load a dropdown with every user in the system?